### PR TITLE
feat(board):Add Heltec Vision Master series boards

### PR DESCRIPTION
**Pull request by RichardHeltec**
Add the new series of development boards launched by Heltec.
Based on the ESP32-S3R8 chip, it integrates displays with various functions and is paired with LoRa modules as an option, allowing users to choose different options for different situations.

[Vision Master E213](https://heltec.org/project/vision-master-e213/)
[Vision Master E290](https://heltec.org/project/vision-master-e290/)
[Vision Master T190](https://heltec.org/project/vision-master-t190/)

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -25574,6 +25574,537 @@ heltec_ht_de01.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_vision_master_e290.name=Heltec Vision Master E290
+heltec_vision_master_e290.vid.0=0x303a
+heltec_vision_master_e290.pid.0=0x1001
```
Comment (me-no-dev) [inline boards.txt:25579]: please remove VID/PID from all board definitions. This is a generic VID/PID of the CDC/JTAG and should exist only once in this file.

Comment (RichardHeltec) [reply boards.txt:25579]: Hello, I have removed VID and PID, please continue to review.

**Comment by RichardHeltec** [comment]
@P-R-O-C-H-Y Hi, dear. we would appreciate it if you could review this PR as soon as possible, because some of our guests are waiting.

**Review by P-R-O-C-H-Y — APPROVED** [review]
LGTM, @RichardHeltec sorry for delay on review, I was on vacation last week.

**Pull request merged**